pkgs/misc/rkdeveloptool/default.nix

```
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation {
   pname = "rkdeveloptool";
-  version = "1.3";
+  version = "1.3-62";
```

```suggestion
  version = "unstable-2019-06-30";
```
This should follow the naming convention for packages that don't use a stable release, see: https://nixos.org/nixpkgs/manual/#sec-package-naming

---

Hm yeah, wasn't sure how to proceed since the application apparently is "stuck" in version 1.3. Will update to the version you proposed.